packages/devextreme/js/__internal/ui/menu/m_menu.ts

```diff
@@ -438,6 +438,7 @@ class Menu extends MenuBase {
       selectNodesRecursive: false,
       selectByClick: this.option('selectByClick'),
       expandEvent: 'click',
+      adaptivityEnabled: this._isAdaptivityEnabled(),
```

Comment:
[excess code]
This code is executed only if `this._isAdaptivityEnabled()` is `true`. So here you can just pass `true`

Reply:
[d45533d](https://github.com/DevExpress/DevExtreme/pull/27647/commits/d45533db3d0fdfffa3fcf4d8a704612c6f429a4e)